packages/vm/tests/api/runTx.spec.ts

```diff
@@ -559,3 +560,28 @@ tape('runTx() -> consensus bugs', async (t) => {
     t.end()
   })
 })
+
+tape('runTx() -> RunTxOptions', (t) => {
+  t.test('should throw on negative value args', async (t) => {
+    for (const txType of TRANSACTION_TYPES) {
+      const vm = new VM({ common })
```

Comment:
another tiny nit but you can move this VM init above the for loop scope to make it a bit faster? probably tiny difference but adds up in the suite over time

Reply:
Yeah good point